src/01-dinosaur-facts.js

```diff
@@ -44,7 +61,19 @@ function getLongestDinosaur(dinosaurs) {}
  *  getDinosaurDescription(dinosaurs, "incorrect-id");
  *  //> "A dinosaur with an ID of 'incorrect-id' cannot be found."
  */
-function getDinosaurDescription(dinosaurs, id) {}
+function getDinosaurDescription(dinosaurs, id) {
+let dinosaurDescription
+
+  for (let dino of dinosaurs){
+    if(dino.dinosaurId === id){
+      dinosaurDescription = `${dino.name} (${dino.pronunciation})\n${dino.info} It lived in the ${dino.period} period, over ${dino.mya[dino.mya.length-1]} million years ago.`
+      return dinosaurDescription
+    }
+  
+  }
+  dinosaurDescription = "A dinosaur with an ID of 'incorrect-id' cannot be found."
```

Comment:
This return can be more dynamic. If the id was `123abcd`, we would want the error message to be return with the appropriate incorrect id